tripy/tripy/utils/utils.py

```diff
@@ -447,22 +448,26 @@ def get_positional_arg_names(func, *args):
         arg_names.append(name)
 
     # For all variadic positional arguments, assign the name of the variadic group.
-    arg_names.extend([varargs_name] * (len(args) - len(arg_names)))
-    return list(zip(arg_names, args))
+    num_variadic_args = len(args) - len(arg_names)
+    variadic_start_idx = len(arg_names)
+    arg_names.extend([varargs_name] * num_variadic_args)
+    return list(zip(arg_names, args)), (varargs_name, variadic_start_idx) if num_variadic_args > 0 else None
 
```

Comment:
I was thinking that we could do something more like:
```py
    variadic_start_idx = None
    for index, (name, param) in enumerate(signature.parameters.items()):
        if param.kind == inspect.Parameter.VAR_POSITIONAL:
            # Positional arguments cannot follow variadic positional arguments
            # (they would just be absorbed into the variadic argument).
            variadic_start_idx = index
            varargs_name = name
            break
```

Reply:
Ah, even simpler